File: src/CSP-AF/CSPCore/CSPServices/CSPOpenBrowser.cs
using System;
using System.Collections.Generic;

using CloudSuitePortal;
using CloudSuitePortal.Services;

class CSPOpenBrowser
{
    public void run(Dictionary<string, object> Params) 
    {
        var url = (string)Params["URL"];
        var browserType = (string)Params["Browser Type"];
        CSPBrowser.GoToURL(url, browserType);
    }
}

File: src/CSPAF/CSPCore/CSPServices/CSPOpenBrowser.java
package CSPAF.CSPCore.CSPServices;

import java.util.*;

import com.infor.cloudsuite.portal.core.services.*;

class CSPOpenBrowser
{
    public void run(HashMap<String, Object> params)
    {
        String url = params.get("URL").toString();
        String browserType = params.get("Browser Type").toString();
        CSPBrowser.to(url, browserType);
    }
}